tests/test_google_analytics_bookmarks.py

```diff
@@ -210,23 +211,26 @@ def test_run(self):
                     #     According to Gooogle Analytics Docs, is_golden should only be false for the last 24-48
                     #     hours of data.
 
+                    # The assertion below is only valid if data is being generated daily.  New data
+                    #   has stopped as of Oct 5, 2023.  The old account is reportedly no longer
+                    #   running.  If access to an active account is possible this can be uncommented
                     # Verify the first sync bookmark falls within the date window of today - 48 hours
-                    today_minus_golden_window = self.timedelta_formatted(today, days=-2)
-                    self.assertGreaterEqual(
-                        first_bookmark_value, today_minus_golden_window,
-                        msg="First sync bookmark was set prior to the expected date."
-                    )
-
+                    # today_minus_golden_window = self.timedelta_formatted(today, days=-2)
+                    # self.assertGreaterEqual(first_bookmark_value, today_minus_golden_window,
+                    #     msg="First sync bookmark was set prior to the expected date."
+                    # )
```

Comment:
Can we remove these commented assertions.
As analytics will not send any new data if my understanding is correct as per discussion in stand-up

Reply:
Yes, I wasn't sure if there was a chance we could get fresh data or not.  Will remove before merging.

Reply:
Decided to keep it for context.